api/src/main/java/org/pmiops/workbench/api/ProfileController.java

```diff
@@ -269,11 +273,20 @@ private User initializeUserIfNeeded() {
         return user;
 
       case ERROR:
-        log.log(Level.SEVERE, String.format(
-            "free tier project %s failed to be created", user.getFreeTierBillingProjectName()));
-        user.setFreeTierBillingProjectStatus(status);
-        return userDao.save(user);
-
+        int retries = Optional.ofNullable(user.getBillingProjectRetries()).orElse(0);
+        if (retries <= 2) {
+          this.userService.setBillingRetryCount(retries + 1);
+          fireCloudService.removeUserFromBillingProject(user.getEmail(), user.getFreeTierBillingProjectName());
+          String billingProjectName = createFirecloudBillingProject(user);
+          user.setFreeTierBillingProjectName(billingProjectName);
+          user.setFreeTierBillingProjectStatus(BillingProjectStatus.PENDING);
+          return userDao.save(user);
+        } else {
+          log.log(Level.SEVERE, String.format(
+              "free tier project %s failed to be created", user.getFreeTierBillingProjectName()));
+          user.setFreeTierBillingProjectStatus(status);
```

Comment:
At this point I guess the user can't use the free tier billing project... which seems ok... the UI will indicate as such?

Reply:
Yeah... I think we need some sort of UI error indicator here. I'm not sure what to do in this situation though.